src/common/tracing/src/config.rs

```diff
@@ -46,7 +46,7 @@ impl Config {
             tracing: TracingConfig {
                 on: true,
```

Comment:
```suggestion
                on: false,
```

Reply:
It has been on for a long time, and meta's unittest needs to report traces.